_inc/client/components/module-settings/index.jsx

```diff
@@ -794,3 +794,52 @@ export let SitemapsSettings = React.createClass( {
 } );
 
 SitemapsSettings = moduleSettingsForm( SitemapsSettings );
+
+export let WordAdsSettings = React.createClass( {
+	render() {
+		return (
+			( this.props.getOptionValue( 'wordads_tos' ) && this.props.getOptionValue( 'wordads_approved' ) ) ?
+			<form onSubmit={ this.props.onSubmit } >
+				<FormFieldset>
+					<ModuleSettingCheckbox
+						name={ 'enable_header_ad' }
+						{ ...this.props }
+						label={ __( 'Display an ad unit at the top of each page.' ) } />
+					<FormButton
+						className="is-primary"
+						isSubmitting={ this.props.isSavingAnyOption() }
+						disabled={ this.props.shouldSaveButtonBeDisabled() } />
+				</FormFieldset>
+			</form>
+			:
+			<div>
+				<p>{ __( 'WordAds still requires the following actions to activate:' ) }</p>
+				<ul>
+					{ ! this.props.getOptionValue( 'wordads_approved' ) ?
```

Comment:
Late during the development of the first Jetpack React we talked with @zingor about adopting the WP Calypso coding guidelines. For example, ternary operator is covered here https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements
```js
var baz = firstCondition( foo ) && secondCondition( bar )
    ? qux( foo, bar )
    : foo;
```
this notation is specially helpful for large blocks and it brings consistency. It probably makes sense to adopt their coding guidelines since we'll be part of it more and more.